services/core-api/app/api/mines/tailings/models/tailings.py

```diff
@@ -40,7 +40,7 @@ class TailingsStorageFacilityType(Enum):
     def __str__(self):
         return self.value
 
-class MineTailingsStorageFacility(AuditMixin, Base):
+class MineTailingsStorageFacility(AuditMixin, HistoryMixin, Base):
```

Comment:
I thought about doing this, but didn't actually do it (because scope I guess?), and I'm glad you did.